src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.Tests/AZC0002Tests.cs

```diff
@@ -160,91 +152,103 @@ namespace RandomNamespace
 {
     public class SomeClient
     {
-        public virtual Task GetAsync(string s)
+        public virtual Task {|AZC0002:GetAsync|}(CancellationToken cancellationToken = default, string text = default)
         {
             return null;
         }
 
-        public virtual void Get(string s)
+        public virtual void {|AZC0002:Get|}(CancellationToken cancellationToken = default, string text = default)
         {
         }
+    }
+}";
+            await Verifier.CreateAnalyzer(code)
+                .WithDisabledDiagnostics("AZC0015")
+                .RunAsync();
+        }
 
-        public virtual Task GetAsync(string s, RequestContext context = default)
+        [Fact]
+        public async Task AZC0002NotProducedForMethodsWithCancellationToken()
+        {
+            const string code = @"
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace RandomNamespace
+{
+    public class SomeClient
+    {
+        public virtual Task GetAsync(CancellationToken cancellationToken = default)
         {
             return null;
         }
 
-        public virtual void Get(string s, RequestContext context = default)
+        public virtual void Get(CancellationToken cancellationToken = default)
         {
         }
     }
 }";
             await Verifier.CreateAnalyzer(code)
                 .WithDisabledDiagnostics("AZC0015")
                 .RunAsync();
-        }
-
+        }
+
         [Fact]
-        public async Task AZC0002ProducedWhenCancellationTokenOverloadsDontMatch()
+        public async Task AZC0002NotProducedForMethodsWithRequestContentAndCancellationToken()
```

Comment:
Is this method named incorrectly shouldn't it be `AZC0002NotProducedForMethodsWithRequestContextAndCancellationToken